src/components/data/paginator/paginator.scss

```diff
@@ -20,6 +20,8 @@
   }
 
   @media screen and (max-width: constants.$breakpoint-desktop - 1px) {
+    width: 100%;
```

Comment:
(No change in this PR, but...): This might beome a challenge for accessibility in future (depending on whether a page loads a new URL or not, etc. And aria-labels such as 'previous' might not have meaning for accessibility + you might need to indicate which page is currently active with something like `aria-current="true"`.

Reply:
As discussed: no change for now.